Gem/Code/Source/Platform/Windows/Utils_Windows.cpp

```diff
@@ -22,19 +22,23 @@ namespace AtomSampleViewer
         {
             bool result = false;
 
-            AZStd::string exe = "C:\\Program Files\\Beyond Compare 4\\BCompare.exe";
-            AZStd::string commandLine = "\"" + exe + "\" \"" + filePathA + "\" \"" + filePathB + "\"";
-
-            STARTUPINFO si;
+            AZStd::wstring exeW = L"C:\\Program Files\\Beyond Compare 4\\BCompare.exe";
+            AZStd::wstring filePathAW;
+            AZStd::to_wstring(filePathAW, filePathA.c_str());
+            AZStd::wstring filePathBW;
+            AZStd::to_wstring(filePathBW, filePathB.c_str());
+            AZStd::wstring commandLineW = L"\"" + exeW + L"\" \"" + filePathAW + L"\" \"" + filePathBW + L"\"";
+
+            STARTUPINFOW si;
```

Comment:
wow paintful to see this burned into the EXE like this, but its not your fault.  Maybe open an issuee to redirect to regset file at least...

Reply:
@o3de/sig-presentation What is this trying to do?
Is it to launch Beyond Compare for the user to see a UI?
Or is it actually trying to compare the file contents?
Because if it is the latter that can be done with two SystemFile objects and a `memcmp`

Reply:
same feeling... this is part of the ASV project so I figured it was ok for them...